src/ontology/mondo-edit.obo

```diff
@@ -582793,6 +582792,17 @@ property_value: http://purl.org/dc/elements/1.1/date "2024-07-01T21:00:19Z" xsd:
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7652" xsd:anyURI
 
+[Term]
+id: MONDO:1040032
+name: EN1-related dorsoventral syndrome
+subset: ordo_disorder {source="Orphanet:611223"}
+synonym: "ENDOVE syndrome" EXACT [Orphanet:611223]
+synonym: "ENDOVES" EXACT [Orphanet:611223]
```

Comment:
are these abbreviations? If so, please add the abbreviation synonym type annotation

Reply:
comment addressed